etc/config/fortran.defaults.properties

```diff
@@ -11,16 +11,28 @@ group.gfortran_86.compilers=gfortran412:gfortran447:gfortran453:gfortran464:gfor
 group.gfortran_86.groupName=GCC x86-64
 compiler.gfortran412.exe=/usr/bin/gfortran-4.1.2
 compiler.gfortran412.name=gfortran 4.1.2
+compiler.gfortran412.supportsBinary=false
+compiler.gfortran412.supportsExecute=false
 compiler.gfortran447.exe=/usr/bin/gfortran-4.4.7
 compiler.gfortran447.name=gfortran 4.4.7
+compiler.gfortran447.supportsBinary=false
+compiler.gfortran447.supportsExecute=false
 compiler.gfortran453.exe=/usr/bin/gfortran-4.5.3
 compiler.gfortran453.name=gfortran 4.5.3
+compiler.gfortran453.supportsBinary=false
+compiler.gfortran453.supportsExecute=false
 compiler.gfortran464.exe=/usr/bin/gfortran-4.6.4
 compiler.gfortran464.name=gfortran 4.6.4
+compiler.gfortran.supportsBinary=false
```

Comment:
`gfortran464`?

Reply:
oh right, nice catch. I messed up and didn't see this is the `default` file, not the amazon one.